ERCS/erc-7902.md

```diff
@@ -0,0 +1,200 @@
+---
+eip: 7902
+title: Wallet Capabilities for Account Abstraction
+description: New Wallet Capabilities providing a way for dApps and Account Abstraction wallets to communicate
```

Comment:
```suggestion
description: EIP-5792 Capabilities providing a way for dApps and Account Abstraction wallets to communicate
```

Specifically mentioning 5792 will make it more obvious where this proposal fits when browsing the list of ERCs.